### DISCUSSION TITLE
Issure trying to run cases_small

### DISCUSSION THREAD

**Discussion by YoumnaAtta0** [NREL/ReEDS-2.0#179] [Q&A: Execution]
I am trying to run the cases_small.csv. However, I keep facing this error and no files are generated. It only creates the directory with empty folders. 
"Exception: reeds2pras_path=~/github/ReEDS2PRAS or Manifest.toml does not exist. Please set up ReEDS2PRAS and julia by following the instructions at https://github.nrel.gov/ReEDS/ReEDS-2.0/wiki/ReEDS2PRAS,-Julia,-and-stress-periods-setup if you haven't already, or point reeds2pras_path to your ReEDS2PRAS repo."

<img width="515" alt="image" src="https://github.com/user-attachments/assets/be36a936-c522-412d-9ec8-01cea0d7886e">

I navigated to https://github.com/NREL/ReEDS2PRAS?tab=readme-ov-file#readme to setup ReEDS2PRAS, but I am facing this error
"(@v1.11) pkg> registry add https://github.nrel.gov/JuliaLang/InternalRegistry.git
     Cloning registry from "https://github.nrel.gov/JuliaLang/InternalRegistry.git"
ERROR: failed to clone from https://github.nrel.gov/JuliaLang/InternalRegistry.git, error: GitError(Code:ERROR, Class:OS, failed to send request: The server name or address could not be resolved"

**Comment by kennedy-mindermann** [comment]
We’re working on publishing a new version of ReEDS that includes ReEDS2PRAS. Once it’s released, you should be able to set up and run ReEDS with `cases_small.csv` successfully.

**Reply by YoumnaAtta0** [reply]
Could you share a case file that I can run using the GAMS demo version or the academic version?

**Reply by wesleyjcole** [reply]
You should be able to run cases_small without ReEDS2PRAS turned on.  Simple set GSw_PRM_CapCredit to be 1 and the model will skip ReEDS2PRAS.

**Reply by YoumnaAtta** [reply]
I have tried running cases_small without ReEDS2PRAS turned on. However, the run still exceeds the GAMS academic license limit. If you could share a case file that can run using the GAMS academic version, that would be great. Thanks.

**Reply by wesleyjcole** [reply]
Would you try changing GSw_Region to be "st/ND" and rerun?  That should shrink the model size by quite a bit--I'm hoping it will do it enough to get runsize down to what will work with the academic license.

**Reply by YoumnaAtta** [reply]
When I try to run the model and setting GSw_Region to be "st/ND", "nd" or "ny", I face a value error. 

Here is the gamslog:
computer,repo,branch,commit,description
Youmnas-Air.cable.rcn.com,/Users/youmna/Documents/GitHub/ReEDS-2.0/.git,main,9b159a182a95bd7ff96bb8fa2f9d1110744fb51a,v2023-13-g9b159a1
copy_files.py | 2024-07-23 10:25:50 | INFO | Starting copy_files.py
copy_files.py | 2024-07-23 10:25:50 | INFO | Copying non-region-indexed files
copy_files.py | 2024-07-23 10:25:50 | INFO | Copying region-indexed files: filtering for valid regions
copy_files.py | 2024-07-23 10:25:52 | INFO | Finished copy_files.py
calc_financial_inputs.py | 2024-07-23 10:25:52 | INFO | Starting calculation of financial parameters for
calc_financial_inputs.py | 2024-07-23 10:25:52 | INFO | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/inputs_case
calc_financial_inputs.py | 2024-07-23 10:25:52 | INFO | df_ivt created for
calc_financial_inputs.py | 2024-07-23 10:25:52 | INFO | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/inputs_case
calc_financial_inputs.py | 2024-07-23 10:25:52 | ERROR | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/input_processing/support_functions.py:183: FutureWarning: Setting an item of incompatible dtype is deprecated and will raise in a future error of pandas. Value '0.9687761353822694' has dtype incompatible with int64, please explicitly cast to a compatible dtype first.
calc_financial_inputs.py | 2024-07-23 10:25:52 | ERROR |   sys_financials.loc[year,'pvf_capital'] = (
calc_financial_inputs.py | 2024-07-23 10:25:53 | INFO | Calculating financial multipliers for
calc_financial_inputs.py | 2024-07-23 10:25:53 | INFO | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/inputs_case
calc_financial_inputs.py | 2024-07-23 10:25:53 | INFO | ...
calc_financial_inputs.py | 2024-07-23 10:25:53 | INFO | Finished calc_financial_inputs.py
fuelcostprep.py | 2024-07-23 10:25:53 | INFO | Starting fuelcostprep.py
fuelcostprep.py | 2024-07-23 10:25:53 | INFO | Finished fuelcostprep.py
writecapdat.py | 2024-07-23 10:25:54 | INFO | Starting writecapdat.py
writecapdat.py | 2024-07-23 10:25:54 | INFO | Importing generator database:
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering non-RSC Existing Capacity...
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering non-RSC Prescribed Capacity...
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering RSC Existing Capacity...
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering RSC Prescribed Capacity...
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering SMR Existing Capacity...
writecapdat.py | 2024-07-23 10:25:54 | ERROR | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/input_processing/writecapdat.py:447: FutureWarning: DataFrame.fillna with 'method' is deprecated and will raise in a future version. Use obj.ffill() or obj.bfill() instead.
writecapdat.py | 2024-07-23 10:25:54 | ERROR |   h2_ba_share.loc[:2021] = h2_ba_share.loc[:2021].fillna(method='bfill')
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering Retirement Data...
writecapdat.py | 2024-07-23 10:25:54 | INFO | Gathering Wind Retirement Data...
writecapdat.py | 2024-07-23 10:25:54 | ERROR | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/input_processing/writecapdat.py:653: FutureWarning: The 'axis' keyword in DataFrame.groupby is deprecated and will be removed in a future version.
writecapdat.py | 2024-07-23 10:25:54 | ERROR |   .groupby('r', axis=0).max()
writecapdat.py | 2024-07-23 10:25:54 | INFO | Writing out capacity data
writecapdat.py | 2024-07-23 10:25:54 | INFO | Finished writecapdat.py
writesupplycurves.py | 2024-07-23 10:25:56 | INFO | Starting writesupplycurves.py
writesupplycurves.py | 2024-07-23 10:25:56 | INFO | DUPV turned off
writesupplycurves.py | 2024-07-23 10:25:56 | ERROR | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/input_processing/writesupplycurves.py:516: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
writesupplycurves.py | 2024-07-23 10:25:56 | ERROR |   alloutcap = pd.concat([windcap, cspcap, upvcap, dupvcap])
writesupplycurves.py | 2024-07-23 10:25:56 | ERROR | /Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/input_processing/writesupplycurves.py:518: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
writesupplycurves.py | 2024-07-23 10:25:56 | ERROR |   pd.concat([windcost, cspcost, upvcost, dupvcost])
writesupplycurves.py | 2024-07-23 10:25:56 | INFO | Finished writesupplycurves.py
writedrshift.py | 2024-07-23 10:25:57 | INFO | Starting writedrshift.py
writedrshift.py | 2024-07-23 10:25:57 | ERROR | Traceback (most recent call last):
writedrshift.py | 2024-07-23 10:25:57 | ERROR |   File "/Users/youmna/Documents/GitHub/ReEDS-2.0/runs/trial7_nd_small/input_processing/writedrshift.py", line 146, in <module>
writedrshift.py | 2024-07-23 10:25:57 | ERROR | evmc_shape_profile_decrease = pd.read_hdf(
writedrshift.py | 2024-07-23 10:25:57 | ERROR |   File "/opt/anaconda3/envs/reeds2/lib/python3.11/site-packages/pandas/io/pytables.py", line 434, in read_hdf
writedrshift.py | 2024-07-23 10:25:57 | ERROR | raise ValueError(
writedrshift.py | 2024-07-23 10:25:57 | ERROR | ValueError
writedrshift.py | 2024-07-23 10:25:57 | ERROR | :
writedrshift.py | 2024-07-23 10:25:57 | ERROR | Dataset(s) incompatible with Pandas data types, not table, or no datasets found in HDF5 file.
writedrshift returned 0

**Reply by kennedy-mindermann** [reply]
It looks like there is a bug in the current version of ReEDS. I would recommend checking out an older version ([v2024.2.0](https://github.com/NREL/ReEDS-2.0/releases/tag/v2024.2.0)) and then running ReEDS.

**Reply by YoumnaAtta** [reply]
I still face the same error when using the older version (v2024.2.0). I run into this problem using pandas 2.0.3 and pandas 2.1.4.

writedrshift.py | 2024-07-24 13:01:36 | ERROR | raise ValueError(
writedrshift.py | 2024-07-24 13:01:36 | ERROR | ValueError
writedrshift.py | 2024-07-24 13:01:36 | ERROR | :
writedrshift.py | 2024-07-24 13:01:36 | ERROR | Dataset(s) incompatible with Pandas data types, not table, or no datasets found in HDF5 file.
trial1_nd_small has finished

**Reply by wesleyjcole** [reply]
Are you using the reeds2 environment when you run?  If not, that will hopefully fix the issue.  If you are not, then would you check to ensure that the .h5 checked out correctly with git lfs?  You can do that by look at inputs\variability\multi_year to see if the files are large (10s to 100s of MB).
